doc/api/esm.md

```diff
@@ -471,35 +459,35 @@ separate cache.
 <i id="esm_experimental_json_modules"></i>
 
 ## JSON modules
+<!--YAML
+added: v12.9.0
```

Comment:
this seems to presume a backport that doesn’t happen in this pr?

Reply:
This info was taken from `cli.md`:

https://github.com/nodejs/node/blob/a6de95182b8d8c5c2745f7f49adb28bf4d910c09/doc/api/cli.md#L225-L228

but actually it seems wrong, it was added back in v12.0.0:

https://github.com/nodejs/node/blob/d896f03578f2312aaae347de3b5a0b26882effc8/doc/api/cli.md#L141-L144

```suggestion
added: v12.0.0
```